test/format/ops.c

```diff
@@ -504,7 +504,7 @@ skip_insert:			if (col_update(tinfo,
 		 */
 		if (!insert) {
 			dir = (int)mmrand(&tinfo->rnd, 0, 1);
-			for (np = 0; np < mmrand(&tinfo->rnd, 1, 8); ++np) {
+			for (np = 0; np < mmrand(&tinfo->rnd, 1, 30); ++np) {
```

Comment:
Were you able to reproduce the issue when run with the old code with this change?